bundle/edu.gemini.oodb.too.window/src/main/java/edu/gemini/too/email/osgi/OsgiEmailConfig.java

```diff
@@ -74,7 +68,12 @@ public Site getSite() {
     }
 
     public URL getEmailConfig() {
-        return getClass().getClassLoader().getResource("/resources/emailConf.xml");
+        // If we are testing, use the testing email configurations.
+        // If we are in production, use the production email configurations.
+        if (Version.current.isTest())
+            return getClass().getClassLoader().getResource("resources/emailConf-test.xml");
```

Comment:
I think this will need a leading "/".

How about something like

```java
final String test = Version.current.isTest() ? "-test" : "";
return getClass().getClassLoader().getResource("/resources/emailConf" + test + ".xml");
```

Reply:
Yes, I think you're right.